common/modules/gcp-project/main.tf

```diff
@@ -137,3 +138,12 @@ resource "google_storage_bucket" "project-tfstate" {
     enabled = "true"
   }
 }
+
+resource "google_storage_bucket" "project-build-logs" {
```

Comment:
Should we specify any lifecycle rules? Clear these logs after 30 or 60 or 90 days maybe?

Based on the answer to that: should we use NEARLINE or COLDLINE storage_class?

Reply:
I think we can safely remove those logs after 90 days. Regarding storage class – considering amount of data going to be stored (maybe 1 GB / year?), I don't think it worth changing default storage class.